src/app.controller.ts

```diff
@@ -9,4 +9,10 @@ export class AppController {
   getHello(): string {
     return this.appService.getHello();
   }
+
+  @Get()
+  get() {
+    console.log('hello there ?')
+    return this.appService.get();
```

Comment:
PullProofAI

The addition of a `console.log` statement in the `get()` method might not be necessary and could clutter logs in production. Consider removing it for cleaner log output.

```diff
-     console.log('hello there ?')
``` 


<!-- This is an auto-generated comment by PullProofAI -->